content/SCALE/SCALETutorials/DataProtection/Replication/ReplicationWithEncryptionSCALE.md

```diff
@@ -0,0 +1,142 @@
+
+5. Go to **Data Protection > Replication Tasks** and click **Add** to open the **Replication Task Wizard**. 
+   You then specify the from and to sources, task name, and set the schedule.
+  
+   Setting options change based on the source selections. Replicating to or from a local source does not requires an SSH connection.
+   
+This completes the general process for all replication tasks.
+{{< /expand >}}
+
+## Creating a Remote Replication Task for an Encrypted Dataset
+
+To streamline creating simple replication tasks use the **Replication Task Wizard** to create and copy ZFS snapshots to another system. 
+The wizard assists with creating a new SSH connection and automatically creates a periodic snapshot task for sources that have no existing snapshots.
+
+If you have an existing replication task, you can select it on the **Load Previous Replication Task** dropdown list to load the configuration settings for that task into the wizard, and then make change such as assigning it a different destination, select encryption options, schedule, or retention lifetime, etc. 
+Saving changes to the configuration creates a new replication task without altering the task you loaded into the wizard.
+This saves some time when creating multiple replication tasks between the same two systems.
+
+{{< include file="/content/_includes/ReplicationCreateDatasetAndAdminHomeDirSteps.md" type="page" >}}
+
+3. Unlock the source dataset and export the encryption key to a text editor such as Notepad.
+   Go to **Datasets** select the source dataset, locate the **ZFS Encryption** widget and unlock the dataset if locked. 
+   Export the key and paste it in any text editor such as Notepad. If you set up encryption to use a passphrase, you do not need to export a key.
+
+4. Go to **Data Protection** and click **Add** on the **Replication Tasks** widget to open the **Replication Task Wizard**. Configure the following settings:
+   
+   ![CreateRemoteReplicationTask](/images/SCALE/22.12/CreateRemoteReplicationTask.png "New Remote Replication Task")
+   
+   a. Select either **On this System** or **On a Different System** on the **Source Location** dropdown list. 
+      If your source is a remote system, select **On a Different System**. The **Destination Location** automatically changes to **On this System**.       
+      If your source is the local TrueNAS SCALE system, you must select **On a Different System** from the **Destination Location** dropdown list to do remote replication. 
+      
+      TrueNAS shows the number snapshots available for replication.
+    
+    b. Select an existing SSH connection to the remote system.
+       Select **Create New** to open the **[New SSH Connection](#configure-a-new-ssh-connection)** configuration screen.
+    
+    c. Browse to the source pool/dataset(s), then click on the dataset(s) to populate the **Source** with the path. 
+       You can select multiple sources or manually type the names into the **Source** field. 
```

Comment:
might want to add after 'manually type.. field.' the following '(separated by commas)'